docs/ldap.md

```diff
@@ -29,11 +29,23 @@ authentication source:
     'example-ldap' => [
         'ldap:Ldap',
 
-        /* The hostname of the LDAP server. */
-        'hostname' => 'ldap.example.org',
+        /* The ldap-extension to use. Defaults to 'ext_ldap' */
+        'extension' => 'ext_ldap',
```

Comment:
Will it even work when you don't have this? I.e. is it a useful config option to present to users?

Reply:
Technically any ldap-extension can be used.  I see that Symfony only has an adapter implementation for ext-ldap, so it's probably fine to remove it.

Reply:
Kept the code, removed the documentation / examples..  this is now a 'hidden' feature